src/supervisor.adoc

```diff
@@ -857,7 +857,7 @@ The definition of the CBZE field is furnished by the Zicboz extension.
 The definitions of the CBCFE and CBIE fields are furnished by the Zicbom
 extension.
 
-#The definition of the CRE field is furnished by the <<section_cheri_disable,{cheri_priv_m_reg_enable_ext}>> extension.#
+The definition of the {CRE} field is furnished by the <<section_cheri_disable,{cheri_priv_m_reg_enable_ext}>> extension.
```

Comment:
I originally had this marked up to show changes to the priv spec but agree we can drop this.